src/layer/water.js

```diff
@@ -64,13 +64,66 @@ export const water = {
       0.3,
       1,
     ],
+    "fill-outline-color": Color.waterFillTranslucent,
+  },
+  source: "openmaptiles",
+  "source-layer": "water",
+};
+
+export const waterLine = {
+  id: "water_line",
+  type: "line",
+  filter: [
+    "match",
+    ["get", "class"],
+    bigRivers,
+    [">=", ["zoom"], 8],
+    mediumRivers,
+    [">=", ["zoom"], 16],
+    "lake",
+    ["all", ["!=", ["get", "intermittent"], 1], [">=", ["zoom"], 8]],
```

Comment:
Unfortunately, there’s no way to distinguish between a small lake and a large one. The Great Lakes conspicuously lack waterlines until zoom level 8. But for smaller lakes near larger bodies of water, such as Lake Okeechobee, this lack of a waterline looks more intentional.

Reply:
It might just be from years of staring at a lack of Great Lakes in the osm coastline, but it looks intentional to me.

The eastern seaboard looks a little worse.

![coastline1](https://user-images.githubusercontent.com/23022/206701915-0e8d1683-37e6-4ab9-9d0b-3b8d29939ad8.png)
![coastline2](https://user-images.githubusercontent.com/23022/206701919-96fe1c2a-f601-43cc-af59-a4481f387267.png)
![coastline3](https://user-images.githubusercontent.com/23022/206701921-a5d08b15-e9d2-486e-8305-371547ed14f1.png)

Reply:
Are you referring to the North Carolina coastline? I’m unsure why none of the waters behind the barrier islands are `ocean` in the vector tiles; it’s inconsistent with similar waters elsewhere along the Eastern Seaboard and Gulf Coast.

Reply:
It's no mystery at all: https://www.openstreetmap.org/way/815122837

Someone pulled a Chesapeake Bay and put the coastline outside.

Reply:
Also, Virginia's maritime boundary is missing there?

Reply:
In #35, we attempted to hide maritime boundaries except for boundaries between two administrative areas. The vector tiles have North Carolina’s boundary over the Atlantic as `maritime: 0`, whereas it’s `maritime: 1` for the other surrounding states.

Reply:
Ah, I see.  Thanks @Mashin6 for fixing this a couple days ago :-D

Reply:
> It might just be from years of staring at a lack of Great Lakes in the osm coastline, but it looks intentional to me.

Yes, at zoom level 3, there’s nothing unseemly about the shoreline being absent from the Great Lakes as with any other lake. However, it does start to feel like something is missing at z6 or so, when the coastline isn’t likely to be within view anymore. I’m of two minds about this discrepancy: either we’re OK with it on the basis that the waterline at low zoom levels only reflects `natural=coastline` in OSM, or it’s annoying and we should come up with a solution at the data level for inland seas.

Reply:
I've been slowly fixing the NC Outer Banks' coastline to be where it's supposed to be.  This style change does a great job of highlighting the coastline is in the wrong place, and I consider that a feature and not a bug.